### PR TITLE
[TOF] update matcher

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
@shahor02 could you please have a look?

**Comment by njacazio** [comment]
@noferini

File: DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h [hunk 1]
```diff
@@ -76,6 +82,9 @@ class MatchInfoTOF
   double mSignal = 0.0;              ///< TOF time in ps
   float mVz = 0.0;                   ///< Vz from TOF match
   int mChannel = -1;                 ///< channel
+  // Hit pattern information
+  bool mHitUpDown = false;    ///< hit pattern in TOF up-down
+  bool mHitLeftRight = false; ///< hit pattern in TOF left-right
 
   ClassDefNV(MatchInfoTOF, 6);
```
Comment (shahor02) [inline DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h]: Could you increment the class version to 7?

**Review by shahor02 — CHANGES_REQUESTED** [review]

File: DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h (hunk 1 above)
Comment (njacazio) [reply DataFormats/Reconstruction/include/ReconstructionDataFormats/MatchInfoTOF.h]: Done!

**Comment by njacazio** [comment]
Hi @shahor02 this should be ready to merge now

**Review by shahor02 — APPROVED** [review]

**Pull request merged**